scripts/migrations/030_obligation_field_update.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/python
+# -----------------------------------------------------------------------------
+# Copyright Siemens AG, 2020. Part of the SW360 Portal Project.
+#
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# This is a manual database migration script. It is assumed that a
+# dedicated framework for automatic migration will be written in the
+# future. When that happens, this script should be refactored to conform
+# to the framework's prerequisites to be run by the framework. For
+# example, server address and db name should be parameterized, the code
+# reorganized into a single class or function, etc.
+#
+# This script removes the "obligationDatabaseIds" from obligation
+# ---------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+import time
+import couchdb
+import json
+from webbrowser import get
+
+# ---------------------------------------
+# constants
+# ---------------------------------------
+
+DRY_RUN = True
+
+COUCHSERVER = "http://localhost:5984/"
+DBNAME = 'sw360db'
+
+couch=couchdb.Server(COUCHSERVER)
+db = couch[DBNAME]
+
+# set fieldName
+deleteFieldName = "obligationDatabaseIds"
+
+# ----------------------------------------
+# queries
+# ----------------------------------------
+
+# get all obligation with field "obligationDatabaseIds"
+obligation_with_obligationDatabaseIds_query = {"selector": {"type": {"$eq": "obligation"},deleteFieldName: {"$exists": True}}}
+
+# ---------------------------------------
+# functions
+# ---------------------------------------
+
+def removeFieldName(qryResult, fieldToBeRemoved, log):
+    print 'Removing field name '+fieldToBeRemoved+' starts'
+    log['Updated obligation fields '+fieldToBeRemoved] = []
+    for entity in qryResult:
+        del entity[''+fieldToBeRemoved+'']
+        if not DRY_RUN:
+            db.save(entity)
+            print 'Removing field name '+fieldToBeRemoved+' done'
```

Comment:
Maybe more info can be added to this statement like Id.
Current O/P
```
Getting all obligation with field obligationDatabaseIds
found 7 obligation with field obligationDatabaseIds in db!
Removing field name obligationDatabaseIds starts
Removing field name obligationDatabaseIds done
Removing field name obligationDatabaseIds done
Removing field name obligationDatabaseIds done
Removing field name obligationDatabaseIds done
Removing field name obligationDatabaseIds done
Removing field name obligationDatabaseIds done
Removing field name obligationDatabaseIds done
```

Reply:
appended the `id`